src/style.css

```diff
@@ -0,0 +1,37 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 > :nth-child(n),
+.stars--1 > :nth-child(n+2),
+.stars--2 > :nth-child(n+3),
+.stars--3 > :nth-child(n+4),
+.stars--4 > :nth-child(n+5),
+.stars--5 > :nth-child(n+6) {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
You can set these props for all the stars, and then just replace image for active for some of the stars

Reply:
yep, that`s a lot shorter)